### ISSUE TITLE
Design CSV syntax

### ISSUE THREAD

**Issue by jessexu20** [jessexu20/privacy-plus#2]
Look into (Google plus api)[https://developers.google.com/+/web/api/rest/latest/people#resource]
and design csv used for data analysis

**Comment by jessexu20** [comment]
Hey... the org is a list ... which I think you should redesign your table...cuz it make no sense to make a list inside a table ...

**Comment by jessexu20** [comment]
so is place lived

**Comment by ghost** [comment]
I have already mentioned in wiki ... I remained them in the table in case later we need to incorporate them. 
                                                                                            "organizations, emails and placeLived are lists, thus there maybe multiple records, I am not sure for current now we should include them"

**Comment by jessexu20** [comment]
exactly... just to make sure that your table design is finished... cuz @FeifeiWang7 and I have already finished with the code in data collection... Once you are settled, we can pull the trigger and collect the data..
